prog/vnet/update_load_balancer.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+class Prog::Vnet::UpdateLoadBalancer < Prog::Base
+  subject_is :vm
```

Comment:
Noted. I will update this one later since now with multiple commits working on different parts of this file, renaming it will require a good amount of work. I'll do that in a secondary pr.